### ISSUE TITLE
Missing parenthesis for typeof test.

### ISSUE THREAD

**Issue by goozak** [DenisLabrecque/jQuery-UI-Fixed-for-3.5.1#1]
`typeof` test should be wrapped in parenthesis for the `not (!)` operator to have expected effect here:
https://github.com/DenisLabrecque/jQuery-UI-Fixed-for-3.5.1/blob/23ba1fd814d60f8e123fdf58c1967bebc8c4660d/jquery-ui-1.12.1.js#L251

like this one:
https://github.com/DenisLabrecque/jQuery-UI-Fixed-for-3.5.1/blob/23ba1fd814d60f8e123fdf58c1967bebc8c4660d/jquery-ui-1.12.1.js#L123

**Comment by DenisLabrecque** [comment]
Hi Bruno:

Using the following quick test --
    <script>
        testing = function test() { }
        console.log(!typeof(testing) === 'function' || false);
    </script>
seems to show that the not operator works just fine without parentheses. Am
I missing something?

Of course, I agree the intent is made clearer with the parenthesis.

**Comment by goozak** [comment]
You only tested one 'side' of the boolean.  If `testing` is NOT a function, it doesn't work.

Try this version:
```
test_func = function() { };
test_num = 5;
console.log('test_func 1 (should be false)', !typeof(test_func) === 'function' || false);
console.log('test_func 2 (should be false)', !(typeof test_func === 'function') || false);
console.log('test_num 1 (should be true)', !typeof(test_num) === 'function' || false);
console.log('test_num 2 (should be true)', !(typeof test_num === 'function') || false);
```

**Comment by DenisLabrecque** [comment]
You're right.

I think the non-identity operator makes this clearer and less prone to mistakes:

    ( typeof(test_num) !== 'function'  || false) // Evaluates to true
    ((typeof test_num  !== 'function') || false) // Also evaluates to true
    ( typeof test_num  !== 'function'  || false) // Also true

**Comment by DenisLabrecque** [comment]
Thank you very much for your feedback @goozak This is fixed 5cd00ad40b95a5210d4bf2c0aeee70d0f1f000c8
